packages/aws-cdk/lib/contextplugins.ts

```diff
@@ -1,10 +1,10 @@
-import { MissingContext } from '@aws-cdk/cx-api';
+import cxapi = require('@aws-cdk/cx-api');
 import { Mode, SDK } from './api';
 import { debug } from './logging';
 import { Settings } from './settings';
 
 export interface ContextProviderPlugin {
-  getValue(scope: string[], args: string[]): Promise<any>;
+  getValue(filter: cxapi.ContextProviderProps): Promise<any>;
```

Comment:
"filter"?

Reply:
how about `provider: cxapi.MissingContext` instead?

Reply:
After moving `account` and `region` into the dictdionary, is there a reason to pass the entire `ContextProviderProps` object anymore?

We can just do:

```ts
getValue(args: {[key: string]: any})
```
right?

Reply:
ahh - a little different than what I first thought, but let me try that.